lib/WasmReader/WasmBinaryOpCodes.h

```diff
@@ -170,7 +171,12 @@ WASM_MEMSTORE_OPCODE(I64StoreMem16, 0x3d, L_IL, true, Js::ArrayBufferView::TYPE_
 WASM_MEMSTORE_OPCODE(I64StoreMem32, 0x3e, L_IL, true, Js::ArrayBufferView::TYPE_INT32_TO_INT64, "i64.store32")
 
 WASM_MISC_OPCODE(MemorySize, 0x3f, I_I, true, "memory.size")
-WASM_MISC_OPCODE(MemoryGrow,    0x40, I_I, true, "memory.grow")
+WASM_MISC_OPCODE(MemoryGrow, 0x40, I_I, true, "memory.grow")
+#ifdef ENABLE_WASM_BULK
+WASM_MISC_OPCODE(MemoryCopy, 0xd1, I_I, true, "memory.copy")
+WASM_MISC_OPCODE(MemoryFill, 0xd2, I_I, true, "memory.fill")
```

Comment:
instead of `I_I`, we should make a new `WASM_SIGNATURE` for void return and I32, I32, I32 parameters